.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Did you want to commit this?

Reply:
It fixes tsconfig problem, which emerges when project is nested in other folder, like in our case.

Reply:
Right, but you could have it locally and git ignored.